xen-bugtool

```diff
@@ -1516,6 +1520,38 @@ def multipathd_topology(cap):
 
     return stdout
 
+def snmp_xs_conf(cap):
+    sed_snmp_xs_conf = ['sed',
+    '-e', 's#\(community *= *\).*#\\1********#',
+    '-e', 's#\(authentication_key *= *\).*#\\1********#',
+    '-e', 's#\(privacy_key *= *\).*#\\1********#',
+    '/etc/snmp/snmp.xs.conf']
+    pipe = Popen(sed_snmp_xs_conf, bufsize=1, stdin=PIPE, stdout=PIPE, stderr=dev_null)
+    stdout, _ = pipe.communicate()
+    return stdout
+
+def snmpd_xs_conf(cap):
+    sed_snmpd_xs_conf = ['sed',
+    '-e', 's#\(com2sec\( \+\S\+\)\{2\} \+\).*#\\1********#',
+    '/etc/snmp/snmpd.xs.conf']
+    pipe = Popen(sed_snmpd_xs_conf, bufsize=1, stdin=PIPE, stdout=PIPE, stderr=dev_null)
+    stdout, _ = pipe.communicate()
+    return stdout
```

Comment:
Ditto, like above:
```suggestion
def filter_snmpd_xs_conf(cap):
    """Filter /etc/snmp/snmpd.xs.conf with the com2sec community field filtered"""
    with open("/etc/snmp/snmpd.xs.conf", "r") as file:
        return re.sub(r"(com2sec(\s+\S+){2}\s+).*", r"\1REMOVED", file.read())
```
Sample output from the current file on `rt-next`:
```py
...
####
# First, map the community name "public" into a "security name"

#       sec.name       source        community
com2sec notConfigUser  default       REMOVED
...
```

Reply:
Updated.